gems/pending/util/miq-password.rb

```diff
@@ -191,10 +189,12 @@ def self.ez_load(filename, recent = true)
       CryptString.new(nil, params[:algorithm], params[:key], params[:iv])
     end
   end
+  private_class_method :ez_load
```

Comment:
Note, there's no such thing as a protected class method so it was always public before... so it's going from public to private.